### ISSUE TITLE
OpenFilesHelper.init() crashes on Linux

### ISSUE THREAD

**Issue by Sciss** [scaled/scaled#12]
The `IS_MACOS` call is irrelevant, because the JVM loads the class first, causing an `InvocationTargetException` from JavaFX apparently:

```
java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.sun.javafx.application.LauncherImpl.launchApplicationWithArgs(LauncherImpl.java:389)
	at com.sun.javafx.application.LauncherImpl.launchApplication(LauncherImpl.java:328)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at sun.launcher.LauncherHelper$FXHelper.main(LauncherHelper.java:767)
Caused by: java.lang.NoClassDefFoundError: com/apple/eawt/OpenFilesHandler
	at scaled.impl.Scaled$.main(Scaled.scala:115)
	at scaled.impl.Scaled.main(Scaled.scala)
	... 11 more
Caused by: java.lang.ClassNotFoundException: com.apple.eawt.OpenFilesHandler
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:335)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 13 more
Exception running application scaled.impl.Scaled
```

The `IS_MACOS` must therefore be performed in a class that doesn't rely on Mac OS X classes itself.

**Comment by samskivert** [comment]
I've been meaning to fix this problem for a long time. I'll do that now.

**Comment by samskivert** [comment]
This is fixed with bae885ef64ea8cb18ac57414dbe28682afd5b922.

It will take an hour or so for the updated mac-open-files depend to make its way to Maven Central.

**Comment by Sciss** [comment]
Thank you, it works now!
